### PR TITLE
Html and web personal website

### PR DESCRIPTION
**Pull request by turbo-91**
hey there, pls check my challenge HTML and the Web - Personal Website

### CODE REVIEW

File: html-and-web/html-and-the-web_personal-website/cv.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CV</title>
+  </head>
+  <body>
+    <header>
+      <section id="nav">
```
Comment (DonCamillo53) [inline html-and-web/html-and-the-web_personal-website/cv.html:10]: use nav element instead of section element

Comment (mgczarnievicz) [inline html-and-web/html-and-the-web_personal-website/cv.html:10]: please use the `nav` element

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Torben Jost</title>
+  </head>
+  <body>
+    <header>
+      <section id="nav">
```
Comment (mgczarnievicz) [inline html-and-web/html-and-the-web_personal-website/index.html:10]: again

**Review by DonCamillo53 — CHANGES_REQUESTED** [review]
I think this is the wrong project - However, I left you a comment

File: html-and-web/html-and-the-web_personal-website/index.html (hunk 2 above)
Comment (turbo-91) [reply html-and-web/html-and-the-web_personal-website/index.html:10]: thanks, resolved!

File: html-and-web/html-and-the-web_personal-website/cv.html (hunk 1 above)
Comment (turbo-91) [reply html-and-web/html-and-the-web_personal-website/cv.html:10]: thanks, resolved!

**Pull request merged**